src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/AmendCourtBookingServiceTest.kt

```diff
@@ -452,7 +448,10 @@ class AmendCourtBookingServiceTest {
   fun `should fail to amend a Birmingham prison court video booking for Risley prison user`() {
     whenever(videoBookingRepository.findById(1)) doReturn Optional.of(courtBooking().withMainCourtPrisonAppointment())
 
-    assertThrows<CaseloadAccessException> { service.amend(1, mock<AmendVideoBookingRequest>(), PRISON_USER_RISLEY) }
+    val booking = mock<AmendVideoBookingRequest>()
+    whenever(booking.bookingType) doReturn uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.BookingType.COURT
+
```

Comment:
Could use an import to shorten the reference to BookingType?

Reply:
Yarp I know.